app/experimenter/experiments/tests/serializers/test_recipe.py

```diff
@@ -200,57 +203,122 @@ def test_serializer_outputs_expected_schema(self):
-        self.assertDictEqual(
-            serializer.data,
-            {
-                "slug": "normandy-slug",
-                "preferences": [{"preferenceName": "browser.pref", "value": 4}],
-            },
+
+        serializer_data = serializer.data
+
+        self.assertEqual(serializer_data["slug"], "normandy-slug")
+
+        self.assertEqual(
+            dict(serializer_data["preferences"][0]),
+            {"preferenceName": "browser.pref", "value": 4},
         )
 
     def test_serializer_outputs_expected_schema_for_bool(self):
         experiment = ExperimentFactory.create(
             type=Experiment.TYPE_ROLLOUT,
             normandy_slug="normandy-slug",
             rollout_type=Experiment.TYPE_PREF,
+        )
+
+        preference = RolloutPreference(
             pref_type=Experiment.PREF_TYPE_BOOL,
             pref_name="browser.pref",
             pref_value="true",
+            experiment=experiment,
         )
+
+        preference.save()
         serializer = ExperimentRecipePrefRolloutArgumentsSerializer(experiment)
-        self.assertDictEqual(
-            serializer.data,
-            {
-                "slug": "normandy-slug",
-                "preferences": [{"preferenceName": "browser.pref", "value": True}],
-            },
+
+        serializer_data = serializer.data
+
+        self.assertEqual(serializer_data["slug"], "normandy-slug")
+
+        self.assertEqual(
+            dict(serializer_data["preferences"][0]),
+            {"preferenceName": "browser.pref", "value": True},
         )
 
     def test_serializer_outputs_expected_schema_for_str(self):
         experiment = ExperimentFactory.create(
             type=Experiment.TYPE_ROLLOUT,
             normandy_slug="normandy-slug",
             rollout_type=Experiment.TYPE_PREF,
+        )
+
+        preference = RolloutPreference(
             pref_type=Experiment.PREF_TYPE_STR,
             pref_name="browser.pref",
             pref_value="a string",
+            experiment=experiment,
         )
+
+        preference.save()
         serializer = ExperimentRecipePrefRolloutArgumentsSerializer(experiment)
-        self.assertDictEqual(
-            serializer.data,
-            {
-                "slug": "normandy-slug",
-                "preferences": [{"preferenceName": "browser.pref", "value": "a string"}],
-            },
+        serializer_data = serializer.data
+
+        self.assertEqual(serializer_data["slug"], "normandy-slug")
+
+        self.assertEqual(
+            dict(serializer_data["preferences"][0]),
+            {"preferenceName": "browser.pref", "value": "a string"},
+        )
+
+    def test_serializer_outputs_expected_schema_for_multi_pref(self):
+
+        experiment = ExperimentFactory.create(
+            type=Experiment.TYPE_ROLLOUT,
+            normandy_slug="normandy-slug",
+            rollout_type=Experiment.TYPE_PREF,
+        )
+
+        preference = RolloutPreference(
```

Comment:
**Opinion:**

I woulda called these preference1 and preference2 but that's just a style thing